api/models.py

```diff
@@ -21,13 +21,15 @@ class Meta:
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+
```

Comment:
Looks like you added double spacing to this file, please revert back to single spacing for consistency.